getting-started/testing-setup.md

```diff
@@ -0,0 +1,220 @@
+          Rails.root.join('spec', 'factories'),
+        ]
+
+        FactoryBot.definition_file_paths = solidus_paths + extension_paths + app_paths
+      end
+    end
+  end
+end
+```
+{% endcode %}
+
+Finally, modify the configuration and create the support file to reference customized methods. This will give you the ability to create/build factories and stub authentication:
+
+{% tabs %}
+{% tab title="spec/rails\_helper.rb" %}
+```ruby
+...
+# Add additional requires below this line. Rails is not loaded until this point!
+require 'spree/testing_support/authorization_helpers'
+require 'spree/testing_support/controller_requests'
+...
+# The following line is provided for convenience purposes. It has the downside
+# of increasing the boot-up time by auto-requiring all files in the support
+# directory. Alternatively, in the individual `*_spec.rb` files, manually
+# require only the support files necessary.
+Dir[Rails.root.join('spec','support','**','*.rb')].sort.each {|f| require f }
+...
+RSpec.configure do |config|
+  ...
+  config.include Spree::TestingSupport::ControllerRequests, spree_controller: true
+end
+```
+{% endcode %}
+{% endcode %}
+{% endtab %}
+
+{% tab title="spec/support/factory\_bot.rb" %}
+```ruby
+RSpec.configure do |config|
+  #...
+  config.include FactoryBot::Syntax::Methods
+end
+```
+{% endcode %}
+{% endtab %}
+{% endtabs %}
+
+## Byebug
+
+[Byebug](https://github.com/deivid-rodriguez/byebug) is debugging tool which will help you investigate and understand code while it executes. There are no installation instructions required other than to simply insert `byebug` method anywhere in the codebase while you run a test or navigate through your development server. This will pause the execution of code where the method was inserted and provide you with a debugging interface in your shell. We highly recommend familiarizing yourself with byebug's [guide](https://github.com/deivid-rodriguez/byebug/blob/master/GUIDE.md) and [commands](https://github.com/deivid-rodriguez/byebug#byebugs-commands) to learn how to get the most out of this powerful tool.
+## Capybara and Webdrivers
+
+[Capybara](https://github.com/teamcapybara/capybara) helps you test web applications by simulating how a real user would interact with your app. This is complimented by the `webdrivers` gem to provide selenium drivers required by Capybara to test multiple browsers. However, some setup is required to utilize the features we will be testing. It is recommended that you follow [Capybara's guide](https://github.com/teamcapybara/capybara#using-capybara-with-rspec) provided by the content creators to properly setup this library.
+
+For reference, our setup file resembled the following:
+
+{% code title="spec/support/capybara.rb" %}
+```ruby
+require 'capybara/rspec'
+require 'selenium/webdriver'
+
+Capybara.default_max_wait_time = 5
+
+Capybara.register_driver :chrome do |app|
+  Capybara::Selenium::Driver.new(app, browser: :chrome)
+end
```

Comment:
I think specs will work even if we remove this line, we are never using this driver.